src/gui/search/searchwidget.h

```diff
@@ -82,6 +82,7 @@ private slots:
 
     void fillCatCombobox();
     void fillPluginComboBox();
+    void displayPage();
```

Comment:
The function name says almost nothing. Although I can't list any short and descriptive names, even a monster like `selectReadyToSearchOrPluginInstallLook()` looks better to me.